server.go

```diff
@@ -530,7 +530,7 @@ func (server *mongoServer) poolShrinker() {
 				}
 			}
 			server.liveSockets = remainSockets
-			stats.conn(-1*end, server.info.Master)
+			stats.shrunkConn(1)
```

Comment:
Shouldn't this be ```end``` instead of 1?

Reply:
Yeah this is definitely a mistake, I will fix this.